geoips_clavrx/_version.py

```diff
@@ -0,0 +1,16 @@
+# file generated by setuptools_scm
+# don't change, don't track in version control
+TYPE_CHECKING = False
+if TYPE_CHECKING:
+    from typing import Tuple, Union
+    VERSION_TUPLE = Tuple[Union[int, str], ...]
+else:
+    VERSION_TUPLE = object
+
+version: str
+__version__: str
+__version_tuple__: VERSION_TUPLE
+version_tuple: VERSION_TUPLE
+
+__version__ = version = '1.12.2a0'
+__version_tuple__ = version_tuple = (1, 12, 2)
```

Comment:
This should be 0.0.0 @jsolbrig

Reply:
This actually was an old version of the file from setuptools. I just udpated this for poetry.